src/middlewares/authentication.js

```diff
@@ -23,3 +24,14 @@ exports.authenticateUser = (req, res, next) => {
   req.user = decodedToken;
   next();
 };
+
+exports.isAuthenticated = (req,res,next) =>{
```

Comment:
this function is a duplicate of the authenticateUser function above. Please remove